saleor/dashboard/widgets.py

```diff
@@ -1,25 +1,26 @@
 from django import forms
 from django.conf import settings
 from django.forms import Textarea, TextInput
-from django_filters.widgets import RangeWidget
+from django_filters.widgets import (
+    DateRangeWidget as df_DateRangeWidget, RangeWidget as df_RangeWidget)
```

Comment:
Just a small styling issue - I think `df_DateRangeWidget` and `df_RangeWidget` look bit "unpythonic". To avoid that we could as well do `from django_filters import widgets` and below use it like `widgets. DateRangeWidget` etc., what do you think?